README.md

```diff
@@ -51,8 +51,8 @@ pip install m2cgen
       </tr>
       <tr>
         <th>Boosting</th>
-        <td>XGBClassifier(gbtree/dart booster only)</td>
-        <td>XGBRegressor(gbtree/dart booster only)</td>
+        <td>XGBClassifier(gbtree/dart booster only), LGBMClassifier</td>
```

Comment:
Please mention in parenthesis that we support gbdt/dart boosters only for LGBM as well.

Reply:
Done

Reply:
`gbtree` is called `gbdt` in lightGBM :)